src/components/MainNav/CurrentApp/CurrentApp.js

```diff
@@ -4,22 +4,45 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import NavButton from '../NavButton';
+import AppIcon from '@folio/stripes-components/lib/AppIcon';
+import Badge from '@folio/stripes-components/lib/Badge';
+import Headline from '@folio/stripes-components/lib/Headline';
+import css from './CurrentApp.css';
 
 const propTypes = {
-  currentApp: PropTypes.object,
+  currentApp: PropTypes.shape(
+    {
+      displayName: PropTypes.string,
+      description: PropTypes.string,
+    },
+  ),
+  id: PropTypes.string,
+  icon: PropTypes.oneOfType([
+    PropTypes.element,
+  ]),
+  badge: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
 };
 
-const CurrentApp = ({ currentApp }) => {
-  const { displayName, description } = currentApp;
+const defaultProps = {
+  currentApp: { displayName: 'FOLIO', description: 'FOLIO platform' },
+};
```

Comment:
Instead of having "Folio" as default it should be the last open app as default (or the first app in the list) meaning that you shouldn't be able to _not_ have any open apps.

I think we can leave it like this for now since the "last open app" feature isn't made yet. The last open app should be stored somewhere - probably on each individual user. It probably needs it's own issue I assume.

Reply:
What is supposed to happen when a user logs into FOLIO for the very first time? I imagine that once the 'last open app' feature is implemented, that app will actually be open on log-in, and thus will send its info to this component's currentApp prop.

Reply:
That's a good question! I have asked Filip for more info but if you check out the sign up prototype: http://ux.folio.org/prototype/en/sign-up

Once you are logged in for the first time it looks like it will send you to the Users-app.

Reply:
Given the current state of things with currentApp coming through as undefined at the start, I feel that static, self-supplied defaults are best for right now. Better than this component reaching outside of itself and leaning on something, or the consequence of there *not being an <h1> tag on the page if it doesn't render. If there are better ideas for acceptable static defaults, perhaps even a better solution to the lack-of-h1 issue, I'm all ears.

Reply:
@rasmuswoelk @JohnC-80  I'd be interested in learning about the first time login use-case as well.  While the home page is sparse in current form, surely there is a case for not having an app open.  This would be the perfect place for dashboard metrics and KPI's.

Reply:
@ttamj I'll talk to the designers and hear what they have to say. I heard about one possibility which is to have a "Welcome"-app of some sorts that is shown to the user the first time they log on to the system but I'm not sure if that's the final solution.

Reply:
@filipjakobsen You probably have more info on this matter.

Reply:
@ttamj Hi Matt,

The need for KPIs etc. will be fulfilled by apps focusing on that. So instead of forcing everyone to use a certain dashboard that we guess will work (and which we would have to maintain centrally), people can make the app they want as a dashboard the app that will open when they open the system. The "To-do" app we are designing will fit this purpose for some users, while a Workflows Dashboard might for others. Still others might want a dashboard app that pulls in widgets from all the different apps, but this would require a complex UI that would function best as a separate app from my point of view.

I think we should simply turn the welcome screen into a "Welcome to FOLIO" app that can in time contain an overview of where to find information, etc. And then have that open when users first log in. It would fit the logic of the general system, and not require any exception logic for when users first log in.

Let me know if you have any questions or thoughts on this.